### PR TITLE
draft security md

### PR DESCRIPTION
**Pull request by jackfarzan**
## Changes in this pull request
Adds security.md

## Checklist
- [ ] This PR represents a single feature, fix, or change.
- [ ] All applicable changes have been documented.
- [ ] Any `TO DO` items (or similar) have been entered as GitHub issues and the link to that issue has been included in a comment.

### CODE REVIEW

**Review by dkozma — APPROVED** [review]
As long as HackerOne is the preferred place to send people for these things, I'm good with this.

**Review by mauricefisher64 — APPROVED** [review]
👍

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Security
+
+C2PA open source repos are maintained in partnership with Adobe. At this time, security policies are maintained solely by Adobe employees.
```
Comment (puhley) [inline SECURITY.md:3]: This C2PA open-source library is maintained in partnership with Adobe. At this time, Adobe is taking point on accepting security reports through their HackerOne portal and public bug bounty program.

**Review by gpeacock — APPROVED** [review]

File: SECURITY.md (hunk 1 above)
Comment (puhley) [inline SECURITY.md:3]: Do we want to list the versions of software that we support? That is sometimes included in security.md files? If it isn't really relevant at this time, we can punt for now.

**Review by DaveStein — APPROVED** [review]

**Pull request merged**